src/p_spec.c

```diff
@@ -2174,7 +2174,18 @@ static void P_SecretRevealed(player_t *player)
 {
   if (hud_secret_message && player == &players[consoleplayer])
   {
-    player->secretmessage = s_HUSTR_SECRETFOUND;
+    if (hud_secret_message == SECRETMESSAGE_COUNT)
+    {
+      static char str_count[32];
+      M_snprintf(str_count, sizeof(str_count), "Secret %d of %d revealed!", player->secretcount, totalsecret);
```

Comment:
Could you break this line, please?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1966/commits/ea80bf1cd7eafd5c8cd86a7d2ff1c9653148f712.